nat-lab/tests/telio.py

```diff
@@ -1158,11 +1142,3 @@ async def collect_core_dumps(self):
                     f" {container_name}:{file_path} {core_dump_destination}"
                 )
                 os.system(cmd)
-
-
-def generate_secret_key() -> str:
```

Comment:
Why were these removed?

Reply:
All this function is doing is calling a function in the bindings, so instead of having this function, the call sites can just call that function from the bindings directly